modules/nw-installing-external-dns-operator-cli.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-external-dns-operator-cli_{context}"]
+= Installing the External DNS Operator by using the CLI
+
+You can install the External DNS Operator by using the CLI.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* You are logged into the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `Namespace` object:
+
+.. Create a YAML file that defines the `Namespace` object:
++
+.Example `namespace.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: external-dns-operator
+----
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file that defines the `OperatorGroup` object:
++
+.Example `operatorgroup.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  upgradeStrategy: Default
+    targetNamespaces:
+  - external-dns-operator
```

Comment:
```suggestion
  upgradeStrategy: Default
  targetNamespaces:
  - external-dns-operator
```

Reply:
ty

Reply:
This one is still having the leading whitespace.